pgjdbc/src/main/java/org/postgresql/ssl/MakeSSL.java

```diff
@@ -20,14 +20,15 @@
 import java.util.logging.Logger;
 
 import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLParameters;
 import javax.net.ssl.SSLSocket;
 import javax.net.ssl.SSLSocketFactory;
 
 public class MakeSSL extends ObjectFactory {
 
   private static final Logger LOGGER = Logger.getLogger(MakeSSL.class.getName());
 
-  public static void convert(PGStream stream, Properties info)
+  public static void convert(PGStream stream, Properties info, boolean directConnect)
```

Comment:
I guess `directConnect` is redundant here. `Properties info` should contain the negotiation anyway.